hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson

```diff
@@ -43,15 +43,22 @@
     }
     {
       name: tl_d_partial_access
-      desc: '''Access CSR with one or more bytes of data
-            For read, expect to return all word value of the CSR
-            For write, enabling bytes should cover all CSR valid fields'''
+      desc: '''Access CSR with one or more bytes of data.
+            For read, expect to return all word value of the CSR.
+            For write, enabling bytes should cover all CSR valid fields.'''
```

Comment:
I don't understand this. It suggests if the TLUL access is for 1 byte it will return more bytes of data. Same thing for writes. Maybe the wording needs clarification?

Reply:
Hey Matute, thank you for raising this question.
Yes I believe this is how our TLUL interface behaves: [TLUL spec](https://docs.opentitan.org/hw/ip/tlul/doc/index.html)

For register write, if we use partial access (Opcode "PutPartialData" with mask and size), this could only work if the mask and size covers all valid fields.
For example, if  status register has 15 fields, each one byte.
Legal: write status register with mask `b'0011`
Illegal: write status register with mask `b'0001` (it does not cover field 9 to 15)

For register read, even if the input size and mask are set to partial read (not all ones), TLUL still returns the full 32 bits value in `d_data`.

And these two rules above are just for registers. Memories follow different protocol rules. This is what I understand.